specs/eventing/data-plane.md

```diff
@@ -1,125 +1,171 @@
-# Knative Eventing Data Plane Contracts
+# Knative Eventing Data Plane Contract
 
 ## Introduction
 
-Developers using Knative Eventing need to know what is supported for delivery to
-user provided components that receive events. Knative Eventing defines contract
-for data plane components and we have listed them here.
-
-## Conformance
+Late-binding event senders and receivers (composing applications using
+configuration) only works when all event senders and recipients speak a common
+protocol. In order to enable wide support for senders and receivers, Knative
```

Comment:
>only works when all event senders and recipients speak a common protocol

We are using cloudevents to _mix_ the protocols with events transferring between in a lossless way. So this is slightly incorrect. CloudEvents is not a protocol, it is a binding onto a protocol.

Reply:
We're using CloudEvents-over-HTTP as the protocol; the CloudEvents protocol, the HTTP protocol, and the binding (mapping) from CloudEvents to HTTP are part of the protocol.